src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14865,8 +14813,6 @@ void VRFCondenserEquipment::VRFOU_PipeLossC(
     using General::SolveRoot;
 
     int TUListNum;      // index to TU List
-    int TUIndex;        // Index to terminal unit
-    int CoilIndex;      // index to coil in terminal unit
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14868]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14869]:(style),[variableScope],The scope of the variable 'CoilIndex' can be reduced.